sample/transforms_test.go

```diff
@@ -0,0 +1,138 @@
+package sample
+
+import (
+	"math"
+	"math/rand/v2"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestTemperature(t *testing.T) {
+	logits, err := Temperature(0.5).Apply([]float64{2, -1, 4, -3, 1, -2, 0})
+	if err != nil {
+		t.Error(err)
+		return
+	}
+	want := []float64{-4, -10, 0, -14, -6, -12, -8}
+	if diff := cmp.Diff(want, logits); diff != "" {
```

Comment:
Use `slices.Equal` instead of the `github.com/google/go-cmp/cmp` dependency here and below

Reply:
cmp comment above ^